src/views/portfolio/projects/ProjectDetailsModal.vue

```diff
@@ -722,6 +732,31 @@ export default {
           this.$toastr.w(this.$t('condition.unsuccessful_action'));
         });
     },
+    async showWarning(message, title) {
+      return new Promise((resolve, reject) => {
+        const WarningConstructor = Vue.extend(NotificationDialog);
+        const instance = new WarningConstructor({
+          propsData: { message, title },
+        });
+        instance.$mount();
+        document.body.appendChild(instance.$el);
+
+        instance.$bvModal.show('notification-dialog');
+        instance.$on('confirm', () => {
+          resolve();
+          this.cleanupInstance(instance);
+        });
+
+        instance.$on('cancel', () => {
+          reject();
+          this.cleanupInstance(instance);
+        });
+      });
+    },
+    cleanupInstance(instance) {
+      document.body.removeChild(instance.$el);
+      instance.$destroy();
+    },
```

Comment:
You might be able to simplify this by using Bootstrap's `msgBoxConfirm`: https://bootstrap-vue.org/docs/components/modal#confirm-message-box